src/components/CreateFunctionBar.js

```diff
@@ -48,72 +49,36 @@ class CreateFunctionBar extends React.Component {
       }
 
     render() {
-        var sessionListItems = [];
-        if (this.props.activeClub) {
-            sessionListItems = this.props.sessionNames.map((name) =>
-                <button className="dropdown-item" key={name} onClick={() => this.props.addSession(name)}>{name}</button>
-            );
-        }
-        const templateListItems = this.props.templateNames.map((name) =>
-            <button className="dropdown-item" key={name} onClick={() => this.props.addTemplate(name)}>{name}</button>
-        );
+        const sessionListItems = this.props.sessionNames.map((name) => ({ text: name, onClick: () => this.props.addSession(name) }));
```

Comment:
I would call these DropdownItems vs ListItems. You already have a list component so it's good to distinguish them.